.github/workflows/bindgen.yml

```diff
@@ -149,6 +149,30 @@ jobs:
           BINDGEN_NO_DEFAULT_FEATURES: ${{matrix.no_default_features}}
         run: ./ci/test.sh
 
+  test_aarch64:
+    name: "Run tests on AArch64"
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: uraimo/run-on-arch-action@v2.0.5
+        name: Run test commands
+        with:
+          arch: aarch64
+          distro: ubuntu20.04
+          env: |
+            LLVM_VERSION: "10.0"
+            BINDGEN_RELEASE_BUILD: 0
+            BINDGEN_FEATURE_RUNTIME: 0
+            BINDGEN_FEATURE_EXTRA_ASSERTS: 0
+            BINDGEN_FEATURE_TESTING_ONLY_DOCS: 0
+            BINDGEN_NO_DEFAULT_FEATURES: 0
```

Comment:
Can we somehow not duplicate all the testing setup? If not, can we document why?

For example, the `curl` command to install rust seems like it should work with `actions-rs/toolchain` instead. If it doesn't, why not?

Reply:
The way `run-on-arch-action` works is by creating a container and the current directory is mounted as a volume within the container. If I installed Rust with `actions-rs/toolchain`, the binaries would not be copied into the container. Does that answer your question?